src/mesh.cpp

```diff
@@ -105,6 +105,8 @@ void readMesh(Mesh &mesh, const std::string &filename, const int dim, const bool
         MPI_Abort(MPI_COMM_WORLD, EXIT_FAILURE);
       }
     }
+  } else {
+    std::cerr << "Connectivity information for " << mesh.fname << "skipped since connectivity is not required.";
```

Comment:
Not an error. Usage of `easylogging` in `mesh.cpp` is not compiled at all on my computer. Also, previous logging information is provided using `std::cerr`. It would be corrected in #103. After #103 all cerr messages inc. errors can be handled properly.